### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by gsouf**
Examples from the readme did not work.
- "props" should be "prop"
- no options for uncompressed i/o, assuming they have been removed?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,29 +18,29 @@ Usage
 Run `ttv --help` to get help, or infer what you can from one of these examples:
 
     # Split CSV file into two sets of a fixed number of rows
-    $ ttv split data.csv --rows=train=9000 --rows=test=1000 --uncompressed-input
+    $ ttv split data.csv --rows=train=9000 --rows=test=1000
 
     # Accepts gzipped data (no flag required). Shorthand argument version. As many splits as you like!
     $ ttv split data.csv.gz --rows=train=65000,validation=15000,test=15000
```
Comment (sd2k) [inline README.md:24]:
```suggestion
    $ ttv split data.csv.gz --rows=train=65000,validation=15000,test=15000 -d
```

It was 4 years ago but it looks like I inverted the flag so it's required for gzipped data and changed from `-u` to `-d`.

File: README.md [hunk 2]
```diff
@@ -18,29 +18,29 @@ Usage
...
+    # Alternatively, specify proportion-based splits.
+    $ ttv split data.csv --prop=train=0.8,test=0.2
 
     # When using proportions, include the total rows to get a progress bar
-    $ ttv split data.csv --props=train=0.8,test=0.2 --total-rows=1234 -u
+    $ ttv split data.csv --prop=train=0.8,test=0.2 --total-rows=1234
 
     # Accepts data from stdin, compressed or not (must give a filename)
     $ cat data.csv | ttv split --rows=test=10000,train=90000 --output-prefix data -u
     $ cat data.csv.gz | ttv split --rows=test=10000,train=90000 --output-prefix data
```
Comment (sd2k) [inline README.md:34]:
```suggestion
    $ cat data.csv.gz | ttv split --rows=test=10000,train=90000 --output-prefix data -d
```

File: README.md [hunk 3]
```diff
@@ -18,29 +18,29 @@ Usage
...
 
     # Using pigz for faster decompression
-    $ pigz -dc data.csv.gz | ttv split --prop=test=0.1,train=0.9 --chunk-size 5000 --output-prefix data -u
+    $ pigz -dc data.csv.gz | ttv split --prop=test=0.1,train=0.9 --chunk-size 5000 --output-prefix data
 
     # Split outputs into chunks for faster writing/reading later
     $ ttv split data.csv.gz --rows=test=100000,train=900000 --chunk-size 5000
```
Comment (sd2k) [inline README.md:40]:
```suggestion
    $ ttv split data.csv.gz --rows=test=100000,train=900000 --chunk-size 5000 -d
```

File: README.md [hunk 4]
```diff
@@ -18,29 +18,29 @@ Usage
...
 
     # Write outputs uncompressed
-    $ ttv split data.csv.gz --prop=test=0.5,train=0.5 --uncompressed-output
+    $ ttv split data.csv.gz --prop=test=0.5,train=0.5
 
     # Reproducible splits using seed
     $ ttv split data.csv.gz --prop=test=0.5,train=0.5 --chunk-size 1000 --seed 5330
```
Comment (sd2k) [inline README.md:46]:
```suggestion
    $ ttv split data.csv.gz --prop=test=0.5,train=0.5 --chunk-size 1000 --seed 5330 -d
```

**Review by sd2k — COMMENTED** [review]
Thanks for catching this and fixing it! It looks like the `-u` flag was removed and made default, and the `-d` flag was added as its inverse, in 475a31400480d3d99bd6cba57d3728dcf6a1cd5. I made some suggestions in the README 👍

**Comment by gsouf** [comment]
@sd2k Thanks for your review. Applied your fixes

**Pull request merged**